### PR TITLE
Minor changes for HTTPD, mount daemons, GXCCD and Zelio driver

### PR DESCRIPTION
**Pull request by karpov-sv**
The only significant one is a fix for computing the precession when compiled with ERFA support (it was not properly taking equation of origins into account). Now seems to work correctly.

### CODE REVIEW

File: src/camd/gxccd.cpp [hunk 1]
```diff
@@ -441,14 +466,23 @@ int GXCCD::startExposure ()
...
+		if (ret == -1)
+			reinitCamera ();
 		return -1;
 	}
 
-	ret = gxccd_start_exposure (camera, getExposure (), getExpType () == 0, getUsedX (), getUsedY (), getUsedWidth (), getUsedHeight ());
+	// The GXCCD library expects pre-binned region coordinates and size
+	ret = gxccd_start_exposure (camera, getExposure (), getExpType () == 0,
+								(int) (getUsedX () / binningHorizontal ()),
+								(int) (getUsedY () / binningVertical ()),
```
Comment (pkubanek) [inline src/camd/gxccd.cpp]: getUsedWidthBinned() etc. will not fix it?

Comment (karpov-sv) [reply src/camd/gxccd.cpp]: There are no binned analogs for getUsedX, so I decided not to add them but just be more verbose in both cases. Should I introduce binned getUsed[XY] versions?

**Comment by pkubanek** [comment]
Yes, please add it. We shall probably change branching strategy - it would be much easier for me to change the code in a branch held at master RTS2 GitHub.On Nov 5, 2018 6:16 PM, Sergey Karpov <notifications@github.com> wrote:@karpov-sv commented on this pull request.



In src/camd/gxccd.cpp:
>  		return -1;
 	}
 
-	ret = gxccd_start_exposure (camera, getExposure (), getExpType () == 0, getUsedX (), getUsedY (), getUsedWidth (), getUsedHeight ());
+	// The GXCCD library expects pre-binned region coordinates and size
+	ret = gxccd_start_exposure (camera, getExposure (), getExpType () == 0,
+								(int) (getUsedX () / binningHorizontal ()),
+								(int) (getUsedY () / binningVertical ()),

There are no binned analogs for getUsedX, so I decided not to add them but just be more verbose in both cases. Should I introduce binned getUsed[XY] versions?

**Comment by karpov-sv** [comment]
Ok, done.

**Pull request merged**